airbyte-cdk/python/setup.py

```diff
@@ -49,8 +49,10 @@
     package_data={"airbyte_cdk": ["py.typed", "sources/declarative/declarative_component_schema.yaml"]},
     install_requires=[
         "airbyte-protocol-models==0.3.6",
+        "avro~=1.11.2",
```

Comment:
can we add these dependencies to the `file-based` and `dev` and dev modules instead of the main? I don't know if we'll have the same issues we had with parquet, but there's no need to add the avro library to all connectors

Reply:
good point will do!